src/js/utils/styles.js

```diff
@@ -626,7 +626,31 @@ export const roundStyle = (data, responsive, theme) => {
       breakpoint &&
       breakpoint.edgeSize[data.size] &&
       (breakpoint.edgeSize[data.size] || data.size);
-    if (data.corner === 'top') {
+    if (data.corners) {
```

Comment:
Nice work here. Was just reading through the discussion. Could we implement the simplified API like Shimi had mentioned by doing a check here that:
```
 if (!('corner' in data)) {
Object.keys(data).forEach(corner => {
...
everything you added
} else {
... existing grommet behavior for backwards compatibility...
}
```

Reply:
Meaning, the user has provided `round` as an object but they haven't used `round.corner`, so they must be wanting the behavior of `round={{ 'top-left': 'small', 'bottom-right': 'medium' }}`

Reply:
Hi taysea,
    Thanks for taking a look at this PR! I think the short answer to your question is yes that would also work. 
In my head the pros of namespacing with "corners" are: 

- Making it clear to users that all the keys in the object are representing corners since the loop is blindly grabbing each key and adding a style for it.
- Allows additional non corner keys to be added (in the root "round" object ) in future API updates without needing to refactor this code to keep backwards compatibility
- Gives "corners" priority over "corner" in case a user provides both in the round object.

I'm open to being mistaken about the above.

That being said, I'm totally fine with changing to what you suggested if you guys disagree about the value of the above "pros".

Reply:
Thanks for listing all of that out. @ShimiSun do you have a preferred direction for the API? Doing `round={{ 'top-left': 'small' }}` OR `round={{ corners: { 'top-left: 'small' } }}` given what is listed above?

Reply:
I like the cleanness of `round={{ 'top-left': 'small' }}` but in order to keep consistency with how `size` and `corner` are currently used within `round` I think we should go with the latter `round={{ corners: { 'top-left: 'small', ... } }}.